### ISSUE TITLE
[Suggestion] Make distrobox work for non-systemd users

### ISSUE THREAD

**Issue by leleleSDX** [89luca89/distrobox#211]
**Is your feature request related to a problem? Please describe.**
currently non-systemd users have to use to [work around](https://github.com/89luca89/distrobox/issues/192#issuecomment-1057927784) to enter the distrobox. 

**Describe the solution you'd like**
Maybe we can find a way to not need to make "/" a shared mount. 

**Describe alternatives you've considered**
Additional config options during create, enter would help

**Comment by 89luca89** [comment]
Hi @leleleSDX 
this is not something distrobox can (and should) fix, but something to be handled by the packagers of your distro. 
If a mount is not shared, it well... cannot be shared between host and container, which is the point of using an highly integrated container like distrobox or toolbox do.

I see on Alpine it works as intended so it's not a systemd vs non-systemd issue, but on how the system mounts are configured by default on your distro

**Comment by anwareset** [comment]
@89luca89 is right, this are not distrobox scope to fix. The work around should be managed by the maintainer of your distro's packages.

**Comment by ncursr** [comment]
I looked into this as I was having the same issue with docker + distrobox on Void Linux.  I looked into how alpine has [packaged](https://git.alpinelinux.org/aports/tree/community/distrobox/APKBUILD) distrobox in their repos and found that they mark podman as a dependency. I also looked into how to [pull from docker hub using podman](https://gist.github.com/isomorphisms/3114ab86960656a729a6b4653001aae2). Then I tested by pulling ubuntu 20.04 from both podman and from distrobox and it works! Hopefully this helps.
